spec/build_spec.rb

```diff
@@ -163,12 +166,14 @@ def initialize
       DummyRelease.new(t_recent, true)
     ]
     @my_branches = [
-      DummyBranch.new(t_too_old, 'a'),
-      DummyBranch.new(t_recent, 'b'),
-      DummyBranch.new(-1, 'c'),
-      DummyBranch.new(t_recent, 'd', true),
-      DummyBranch.new(t_recent, 'e', false, true),
-      DummyBranch.new(t_recent, 'fixes-#191-dialog')
+      DummyBranch.new(t_too_old, 'a', 'commit_message'),
+      DummyBranch.new(t_recent, 'b', 'commit_message'),
+      DummyBranch.new(-1, 'c', 'commit_message'),
+      DummyBranch.new(t_recent, 'd', 'commit_message', true),
+      DummyBranch.new(t_recent, 'e', 'commit_message', false, true),
+      DummyBranch.new(t_recent, 'fixes-#191-dialog', 'commit_message'),
+      DummyBranch.new(t_recent, 'g', "[decent_ci_skip]"),
+      DummyBranch.new(t_recent, 'develop', "[decent_ci_skip]")
```

Comment:
I created two new branches for unit test, both with the skip flag, but one of them is the develop branch.